ci/check-params-env.sh

```
@@ -56,8 +58,8 @@ SIZE_ABSOLUTE_TRESHOLD=100
 function check_variables_uniq() {
     local env_file_path_1="${1}"
     local env_file_path_2="${2}"
-    local allow_value_duplicity="${3:=false}"
-    local is_params_env="${4:=false}"
+    local allow_value_duplicity="${3:-false}"
+    local is_params_env="${4:-false}"
```

Shellcheck (0.11.0)

[warning] is_params_env appears unused. Verify use (or export if used externally).

(SC2034)